src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/LanguageServices/CommandLineArguments.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using Microsoft.CodeAnalysis;
+
+namespace Microsoft.VisualStudio.ProjectSystem.LanguageServices
+{
+    public class CommandLineArguments
```

Comment:
I need to expose the constructors so F# can create these types that are still used after this PR is merged.  I originally looked at opening up the entire `Microsoft.CodeAnalysis.CommandLineArguments` type but that quickly snowballed (Koala-balled?  I'm having trouble coming up with an Australian version of this :smile:) into exposing way more stuff than we probably ever wanted too.